scrapperFunctions/index.js

```diff
@@ -120,7 +85,7 @@ const getRows = (selector) => {
 // eslint-disable-next-line space-before-function-paren
 ScrapPage.prototype.expectedRows = async function (select) {
   try {
-    const rowsQ = await this.page.evaluate(getRows, select);
+    const rowsQ = await this.page.evaluate(ROWS_FROM_TABLE.getRows, select);
```

Comment:
if follow the comment behind, you can applay only the function
```suggestion
    const rowsQ = await this.page.evaluate(getRows, select);
```